kubernetes/fluentd-configmap.yaml

```diff
@@ -171,135 +171,6 @@ data:
         process_start_timestamp ${record["process_start_timestamp"].to_i}
       </record>
     </filter>
-  output.conf: |-
```

Comment:
Note that you could add multiple configmap objects to the same YAML file by using the `---` separator, e.g.:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: fluentd-gcp-config
…
data:
  containers.input.conf: |-
    …
---
kind: ConfigMap
apiVersion: v1
  name: fluentd-gcp-main-config
…
data:
  google-fluentd.conf: |-
    …
```
That might make things simpler…

Reply:
Thanks, merged the files together and successfully tested, PTAL.